app/src/main/java/org/mifos/mobile/utils/NotificationComparator.kt

```diff
@@ -9,9 +9,10 @@ import java.util.*
 class NotificationComparator : Comparator<MifosNotification> {
     override fun compare(mifosNotification1: MifosNotification, mifosNotification2: MifosNotification): Int {
         return when {
+            // comparator function logic to sort notifications in the descending order of their timeStamp :
             mifosNotification2.timeStamp < mifosNotification1.timeStamp -> -1
-            mifosNotification1.timeStamp > mifosNotification2.timeStamp -> 1
+            mifosNotification1.timeStamp < mifosNotification2.timeStamp -> 1
```

Comment:
Great catch! However, both comparators are the same in line number 13 and 14, it is difficult to understand it for developers (although it's technically correct). Let's rename `mifosNotification1` variable to something like `firstNotification` and the other one to `secondNotificaiton`
then do the comparison like:
 `firstNotification.timpeStamp > secondNotificaiton.timeStamp` and 
`firstNotification.timpeStamp < secondNotificaiton.timeStamp `

Reply:
Thank you @jawidMuhammadi, I made the required changes : )